src/client/pages/CountryHome/Collaborators/UserList/UserInvitations/Buttons/Information/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,76 @@
+import './Tooltip.scss'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { Routes } from 'meta/routes'
+import { UserInvitationSummary } from 'meta/user'
+
+import { useAssessment, useCycle } from 'client/store/assessment'
+import { useToaster } from 'client/hooks/useToaster'
+import Icon from 'client/components/Icon'
+
+import { useResendInvitation } from '../../hooks/useResendInvitation'
+
+type Props = {
+  invitationSummary: UserInvitationSummary
+  onClose: () => void
+}
+
+const Tooltip: React.FC<Props> = (props: Props) => {
+  const { invitationSummary, onClose } = props
+  const { uuid: invitationUuid, lang } = invitationSummary
+  const assessment = useAssessment()
+  const cycle = useCycle()
+  const { t } = useTranslation()
+  const { toaster } = useToaster()
+
+  const assessmentName = assessment.props.name
+  const cycleName = cycle.name
+
+  const { resendInvitation, isLoading } = useResendInvitation({
+    invitationSummary,
+    callback: onClose,
+  })
+
+  const url = `${window.location.origin}${Routes.LoginInvitation.generatePath(
+    { assessmentName, cycleName },
+    { invitationUuid, lang }
+  )}`
+
+  return (
+    <div className="invitation-info-box">
```

Comment:
This file is unchanged from previous implementation
It is simply moved to where it is used.